### ISSUE TITLE
[NBug] The specified path, file name, or both are too long. The fu...

### ISSUE THREAD

**Issue by juliandavidmr** [gitextensions/gitextensions#6665]
## Error Details
```
System.IO.PathTooLongException: The specified path, file name, or both are too long. The fully qualified file name must be less than 260 characters, and the directory name must be less than 248 characters.
   at System.IO.PathHelper.GetFullPathName()
   at System.IO.Path.LegacyNormalizePath(String path, Boolean fullCheck, Int32 maxPathLength, Boolean expandShortPaths)
   at System.IO.Path.GetFullPathInternal(String path)
   at GitCommands.FullPathResolver.Resolve(String path)
   at GitUI.Editor.FileViewer.<ShowOrDeferAsync>g__GetFileLength|88_0(<>c__DisplayClass88_0& )
   at GitUI.Editor.FileViewer.ViewFileAsync(String fileName, Action openWithDifftool)
   at GitUI.CommandsDialogs.FormCommit.<SetSelectedDiffAsync>d__129.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at GitUI.CommandsDialogs.FormCommit.<>c__DisplayClass128_0.<<ShowChanges>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at GitUI.ThreadHelper.<>c__DisplayClass13_0.<<FileAndForget>b__0>d.MoveNext()
```

## Environment
- Git Extensions 3.1.0.5877
- Build bb1687f6bd0450d0c1a2bae10a37c65493fc1294
- Git 2.21.0.windows.1
- Microsoft Windows NT 10.0.17134.0
- .NET Framework 4.7.3221.0
- DPI 96dpi (no scaling)

**Comment by RussKie** [comment]
Please fill in all required information and ping us back to re-open.

Windows doesn't support path longer than 260 chars, you could enable git longpaths support, but it is a partial remedy: 
`git config --system core.longpaths true`

**Comment by pmiossec** [comment]
And as a workaround, you could also try to move your repository folder
higher in the file system tree.

**Comment by mstv** [comment]
@juliandavidmr, could you test the [canary build 3.2.0.6014](https://ci.appveyor.com/api/buildjobs/iqqocv9lt59t3mv6/artifacts/Setup%2FGitExtensions-Portable-3.2.0.6014.zip), please?

**Comment by gerhardol** [comment]
Duplicate of #6235 
Assumed, as this is potentially fixed by #6702 the duplicate label is set
